must-gather/README.md

```diff
@@ -0,0 +1,117 @@
+# OpenShift Custom Metric Autoscaler Operator Must-Gather
+`custom-metric-autoscaler-must-gather` is a tool built on top of [OpenShift must-gather](https://github.com/openshift/must-gather)
+that expands its capabilities to gather specific information for the OpenShift Custom Metric Autoscaler Operator.
+
+## Usage
+To gather only Openshift Custom Metric Autoscaler Operator information use the following command: 
+```sh
+$ oc adm must-gather --image=$(oc -n openshift-keda get deployment.apps/custom-metrics-autoscaler-operator \
+    -o jsonpath='{.spec.template.spec.containers[?(@.name == "custom-metrics-autoscaler-operator")].image}')
+```
+
+To gather default [OpenShift must-gather](https://github.com/openshift/must-gather) in addition to Openshift Custom
+Metric Autoscaler Operator information you can combine both images with the following command:
+```sh
+oc adm must-gather \
+  --image-stream=openshift/must-gather \
+  --image=registry.redhat.io/custom-metrics-autoscaler-tech-preview/custom-metrics-autoscaler-rhel8-operator
```

Comment:
Does it make sense to push the operator image to Quay? So that the must-gather command may look like this:

```
oc adm must-gather \
  --image-stream=openshift/must-gather \
  --image=quay.io/openshift/origin-custom-metrics-autoscaler-operator
```

Reply:
I prefer that we use the image in cluster for the docs. That way it will work for any cluster with CMA available, including disconnected ones (assuming they have mirrored the CMA images).

So my preference would be either a two-step:
```
IMAGE="$(oc get packagemanifests -n openshift-marketplace openshift-custom-metrics-autoscaler-operator -o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc.annotations.containerImage}')"
oc adm must-gather --image="$IMAGE"
```

or a one-step:
```
oc adm must-gather --image="$(oc get packagemanifests -n openshift-marketplace openshift-custom-metrics-autoscaler-operator -o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc.annotations.containerImage}')"
```
```

Reply:
Agreed, I lean towards leaving only the [first](https://github.com/openshift/custom-metrics-autoscaler-operator/pull/21/files#diff-f22cb59a526d4b8f524cafa3abb53f4ced4afc9b32ef3b3a420ed87e4b870d2eR8) alternative using the container image of the existing deployment.

Reply:
The issue there is that the deployment won't exist if the operator failed to install. The packagemanifests version should work in any situation as long as the catalog has the entry.

Reply:
True.

Reply:
> IMAGE="$(oc get packagemanifests -n openshift-marketplace openshift-custom-metrics-autoscaler-operator -o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc.annotations.containerImage}')"
oc adm must-gather --image="$IMAGE"

@joelsmith  As per above, can we assume that using the `stable` channel is always safe?

Reply:
Yes. It's the only channel we have 😄